PythonAPI/carla/source/carla/libcarla.pyi

```diff
@@ -0,0 +1,5403 @@
+
+        + Note: If no tick is received in synchronous mode, the simulation will freeze. Also, if many ticks are received from different clients, there may be synchronization issues. Please read the docs about synchronous mode to learn more.
+
+        Args:
+            `seconds (float, optional)`: Maximum time the server should wait for a tick (meters). Defaults to 10.0.\n
+
+        Returns:
+            `int`\n
+        """
+
+    def try_spawn_actor(self, blueprint: ActorBlueprint, transform: Transform, attach_to: Actor = None, attachment=AttachmentType.Rigid) -> Actor:
+        """Same as `spawn_actor()` but returns `None` on failure instead of throwing an exception.
+
+        Args:
+            `blueprint (ActorBlueprint)`: The reference from which the actor will be created.\n
+            `transform (Transform)`: Contains the location and orientation the actor will be spawned with.\n
+            `attach_to (Actor, optional)`: The parent object that the spawned actor will follow around. Defaults to None.\n
+            `attachment (AttachmentType, optional)`: Determines how fixed and rigorous should be the changes in position according to its parent object. Defaults to AttachmentType.Rigid.\n
+
+        Returns:
+            `Actor`\n
+        """
+
+    def unload_map_layer(self, map_layers: MapLayer):
+        """Unloads the selected layers to the level. If the layer is already unloaded the call has no effect.
+
+        + Warning: This only affects "Opt" maps. The minimum layout includes roads, sidewalks, traffic lights and traffic signs.
+
+        Args:
+            `map_layers (MapLayer)`: Mask of level layers to be unloaded.\n
+        """
+
+    def wait_for_tick(self, seconds=10.0) -> WorldSnapshot:
+        """This method is used in asynchronous mode. It makes the client wait for a server tick. When the next frame is computed, the server will tick and return a snapshot describing the new state of the world.
+
+        Args:
+            `seconds (float, optional)`: Maximum time the server should wait for a tick (meters). Defaults to 10.0.\n
+
+        Returns:
+            `WorldSnapshot`\n
+        """
+    # endregion
+
+    # region Getters
+    def get_actor(self, actor_id: int) -> Actor:
+        """Looks up for an actor by ID and returns `None` if not found."""
+
+    def get_actors(self, actor_ids: list = None) -> ActorList:
+        """Retrieves a list of `carla.Actor` elements, either using a list of IDs provided or just listing everyone on stage. If an ID does not correspond with any actor, it will be excluded from the list returned, meaning that both the list of IDs and the list of actors may have different lengths."""
+
+    def get_blueprint_library(self) -> list[BlueprintLibrary]:

```

Comment:
This returns just a BlueprintLibrary and not a list.
Wording is a bit ambiguous in in the original documentation.

https://carla.readthedocs.io/en/latest/python_api/#carla.World